Project EFT/Views/Home/Index.cshtml

```diff
@@ -38,17 +39,28 @@
                                 <th>@sub.SubmissionDate</th>
                                 @if (sub.IsCorrect)
                                 {
-                                    <th>Correct</th>
+                                    <th class="success">Correct</th>
                                 }
                                 else
                                 {
-                                    <th>Incorrect</th>
+                                    <th class="error">Incorrect</th>
                                 }
                             </tr>
+
+                            if (session.ContainsKey("problem" + i + "error")) 
+                            {
+                                clearErrorMessage = session.GetString("problem" + i + "error");
+                            }
                         }
 
                     </table>
+                    <button onclick="window.location.href='/Problem/WipeProblemSubmissions?problemID='+i">Wipe submissions for problem @i</button>
+
```

Comment:
should be "window.location.href='/Problem/WipeProblemSubmissions?problemID='+@i"
Without the @, its just an onclick button that does nothing on that page.  The other page works in wiping the data, but returns a 405 error